tests/unit/test_api_put.py

```diff
@@ -5,9 +5,13 @@
 import pytest
 from structlog.testing import capture_logs
 
-from tests.unit.sample_data import SAMPLE_CONTACTS
+from tests.unit.conftest import SAMPLE_CONTACT_PARAMS
 from tests.unit.test_api import _compare_written_contacts
 
+POST_TEST_CASES = pytest.mark.parametrize(
+    "put_contact", SAMPLE_CONTACT_PARAMS, indirect=True
+)
+
```

Comment:
```suggestion
PUT_TEST_CASES = pytest.mark.parametrize(
    "put_contact", SAMPLE_CONTACT_PARAMS, indirect=True
)

```

Reply:
Good catch 👍🏻 [7bfc447](https://github.com/mozilla-it/ctms-api/pull/628/commits/7bfc4477cfe32f5e1066086e3abc1f809df69304)